libs/ngx-mime/src/lib/test/testAltos.ts

```diff
@@ -112,3 +112,1219 @@ export let testAlto = `<?xml version="1.0" encoding="UTF-8"?>
 	</Layout>
 </alto>
 `;
+
+export const testAltoWithSubsContent = `
```

Comment:
We probably don't need the whole alto file just to test subs content. Should maybe shortened?